.pipelines/java_packaging.yml

```diff
@@ -0,0 +1,99 @@
+parameters:
+- name: JavaPackagingandPublishing
+  displayName: Package and Publish Java Package
+  type: boolean
+  default: true
+stages:
+- stage: Java_Packaging_and_Publishing
+  jobs:
+
+  - job: Windows_CPU_Java_Packaging
+    workspace:
+      clean: all
+    pool: {name: 'onnxruntime-Win-CPU-2022'}
+
+    steps:
+    - script: |
+        call .\build.bat -DOCOS_BUILD_JAVA=ON
+      displayName: build the extensions java package
+
+    - task: CopyFiles@2
+      displayName: 'Copy Java Files for Windows Arch to Artifact Staging Directory'
+      inputs:
+        SourceFolder: 'out\Windows\java\build\libs'
+        TargetFolder: '$(Build.ArtifactStagingDirectory)'
+
+    - task: PublishPipelineArtifact@1
```

Comment:
where is this  package published?

Reply:
under "Published artifacts" of the pipeline for that specific run: https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=409718&view=artifacts&pathAsName=false&type=publishedArtifacts

you can read more here: https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/publish-pipeline-artifact-v1?view=azure-pipelines